modules/tensor_mechanics/include/postprocessors/CriticalTimeStep.h

```diff
@@ -39,6 +39,9 @@ class CriticalTimeStep : public ElementPostprocessor, public GuaranteeConsumer
   /// Density of the material
   const MaterialProperty<Real> & _material_density;
 
+  /// Density of the material
+  const MaterialProperty<Real> * _density_scaling;
```

Comment:
`_scaled_material_density`? The current name makes it sound like a scaling factor

Reply:
Hm, I see, this is only the additional density.